testing/cpm/cpm_nvbench-conda.cmake

```diff
@@ -0,0 +1,57 @@
+#=============================================================================
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include(${rapids-cmake-dir}/cpm/init.cmake)
+include(${rapids-cmake-dir}/cpm/rmm.cmake)
+include(${rapids-cmake-dir}/cpm/nvbench.cmake)
+
+enable_language(CUDA)
+enable_language(CXX)
+
+include(${rapids-cmake-dir}/cuda/set_architectures.cmake)
+rapids_cuda_set_architectures(RAPIDS)
+
+# Make sure that nvbench can build static fmt from inside
+# a conda env
+set(ENV{CONDA_BUILD} "1")
+set(ENV{BUILD_PREFIX} "/usr/local/build_prefix")
+set(ENV{PREFIX} "/opt/local/prefix")
+set(ENV{CONDA_PREFIX} "/opt/conda/prefix")
+set(NVBench_ENABLE_CUPTI OFF)
+rapids_cpm_init()
+rapids_cpm_nvbench()
+
+file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/use_fmt.cpp" [=[
```

Comment:
This is about the smallest we can make the repro. It needs to do linking so that we verify that the changes dropped `nvbench/public_fmt_dep_in_conda.diff` aren't lost from upstream